src/webview/main-refactored.ts

```
@@ -0,0 +1,563 @@
+ */
+
+import { Terminal } from 'xterm';
+import { FitAddon } from 'xterm-addon-fit';
+import { WebLinksAddon } from 'xterm-addon-web-links';
+
+// 型定義のインポート
+import type { WebviewMessage, VsCodeMessage, TerminalConfig } from './types/events.types';
+import type { TerminalSettings } from './types/terminal.types';
+
+// 定数のインポート
+import { TERMINAL_CONSTANTS, THEME_CONSTANTS } from './constants';
```

ESLint

[error] 'THEME_CONSTANTS' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)